docs/usage-guide.md

```diff
@@ -49,6 +49,8 @@ otherwise use the set value.
 
   * `bandwidth_size` Required. Specifies the bandwidth size.
 
+  * `bandwidth_name` Required when share_type is PER. Specifies the bandwidth name and is mandatory for Dedicated Loadbalancer if share_type is per.
```

Comment:
```suggestion
  * `bandwidth_name` Optional, it is required when `share_type=PER`. Specifies the bandwidth name.
```

Reply:
done in two commits as i read your message later